ibis/backends/duckdb/tests/test_geospatial.py

```diff
@@ -306,3 +306,21 @@ def test_literal_geospatial_inferred(con, shp, expected, snapshot):
 def test_load_geo_example(con):
     t = ibis.examples.zones.fetch(backend=con)
     assert t.geom.type().is_geospatial()
+
+
+@pytest.mark.parametrize(
+    ("op", "right", "expected"),
+    [
+        param("length", [], 2, id="GeoSpatialUnOp"),
+        param("distance", [shapely.Point(0, 0)], 0, id="GeoSpatialBinOp"),
+    ],
+)
+def test_ops_geo_literals(con, op, right, expected):
```

Comment:
It just occurred to me that we may want to use a new connection here since we have no guarantee that this geospatial test ran before other ones - for this test we really want to ensure that `load_extenstion("spatial")` hasn't been run for this connection yet. So in that case we don't want to use the `con` fixture here but instead create a new `con` in the test.